mainnet-1/specs/movementt.json

```diff
@@ -0,0 +1,110 @@
+{
+    "proposal": {
+        "title": "Add Specs: Movement Testnet",
+        "description": "Adding new specification support for relaying Movement testnet bardock data on Lava",
+        "specs": [
+            {
+                "index": "MOVEMENTT",
+                "name": "movement testnet bardock",
+                "enabled": true,
+                "imports": [
+                    "APT1"
+                ],
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 5,
+                "blocks_in_finalization_proof": 5,
+                "average_block_time": 1000,
+                "allowed_block_lag_for_qos_sync": 50,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "5000000000"
+                },
+                "api_collections": [
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "rest",
+                            "internal_path": "",
+                            "type": "GET",
+                            "add_on": ""
+                        },
+                        "apis": [
+                            {
+                                "name": "/transactions/wait_by_hash/{txn_hash}",
```

Comment:
if you import movement spec you dont need this one

Reply:
yes.